stable/fluent-bit/values.yaml

```diff
@@ -29,6 +29,12 @@ backend:
 
 env: []
 
+## ConfigMap override where fullname is {{.Release.Name}}-{{.Values.configMapOverrideName}}
+## Defining configMapOverrideName will cause templates/config.yaml
+## to NOT generate a ConfigMap resource
+##
+configMapOverrideName: ""
```

Comment:
I would call it `existingConfigMap`. We have it like this in other charts, usually for secrets. Thus should then serve as the complete name without additionally prefixing it with the release.